src/net/server/mod.rs

```diff
@@ -127,9 +127,9 @@ impl IntoResponse for MpcHelperServerError {
     }
 }
 
-/// Provides a mapping of [`QueryId`]s to senders that forward data to a [`Network`]. Every time a
-/// new query is started, a [`Network`] is created to handle communication for that query. When the
-/// server receives a request, it must know which [`Network`] to forward that request to, so it
+/// Provides a mapping of [`QueryId`]s to senders that forward data to a `Network`. Every time a
```

Comment:
If we're going to be consistent about removing square braces, let's remove these as well.

Reply:
I kept them around. Changed the names